recipes-wpe/wpeframework/include/cobalt.inc

```diff
@@ -0,0 +1,22 @@
+WPE_COBALT_STARBOARD_CONFIGURATION_INCLUDE ?= ""
+WPE_COBALT_STARBOARD_CONFIGURATION_INCLUDE_rpi ?= "third_party/starboard/wpe/rpi/configuration_public.h"
+WPE_COBALT_STARBOARD_CONFIGURATION_INCLUDE_brcm ?= "third_party/starboard/wpe/brcm/arm/configuration_public.h"
```

Comment:
Does that mean we can't build cobalt for anything that is not Broadcom or RPI? If so we might add a machine applicable filter (not sure if that is supported in morty)

Reply:
Yes, currently the cobalt support only in RPI and BRCM. For other platform the cobalt package need to be ported. So will check the possibility off using machine filter